File: week1/1.pl
#!/usr/bin/env perl
use v5.26;
use strict;
use warnings;

my $orig      = 'Perl Weekly Challenge';
my (@matches) = $orig =~ m/e/g;
my $new       = $orig =~ s/e/E/gr;

say $new;
say "Number off 'e' characters: " . scalar(@matches);

File: week1/1.py
#!/usr/bin/env python

# Challenge:
#   Write a script to replace the character ‘e’ with ‘E’ in the
#   string ‘Perl Weekly Challenge’. Also print the number of times
#   the character ‘e’ is found in the string.

orig    = 'Perl Weekly Challenge'
matches = orig.count('e')
new     = orig.replace('e', 'E')

print(new)
print("Number of 'e' characters: " + str(matches))
